app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt

```diff
@@ -9,14 +9,25 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
 import com.kodeco.android.coordplot.ui.theme.MyApplicationTheme
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
-            MyApplicationTheme {
-                PlotSurface()
+            val navController = rememberNavController()
+            NavHost(navController, startDestination = "splash_screen") {
+                composable("splash_screen") {
+                    SplashScreenWithNavigation(navController)
```

Comment:
As an alternative, you can utilize a theme to apply the splash screen, that way you don't have to make a full compose screen  and require a LaunchedEffect with a delay.. That said, this is still a good preparation for our week on Navigation, so nice work here!